### PR TITLE
missing "of"

### PR DESCRIPTION
**Pull request by mindspillage**
exactly what it says on the tin

### CODE REVIEW

**Comment by mattl** [comment]
So, you won't be able to apply this one directly, because we moved files around, but I think this is a legit typo.

File: website TOS [hunk 1]
```diff
@@ -6,7 +6,7 @@ Additional terms: In addition to the Master Terms, your use of any Services may
 Collectively, the Terms: The Master Terms, together with any Additional Terms, form a binding legal agreement between you and Creative Commons in relation to your use of the Services. Collectively, this legal agreement is referred to below as the “Terms.”
 >Human-readable summary of Sec 1: These terms, together with any special terms for particular websites, create a contract between you and Creative Commons. The contract governs your use of all websites operated by Creative Commons, unless a particular website indicates otherwise. These human-readable summaries of each section are not part of the contract, but are intended to help you understand its terms.
 2. Your Agreement to the Terms
-YOUR ACCESS TO OR USE OF ANY THE SERVICES (INCLUDING THE LICENSES, PUBLIC DOMAIN TOOLS, AND CHOOSERS) SIGNIFIES THAT YOU HAVE READ, UNDERSTOOD, AND AGREED TO BE BOUND BY THE TERMS. By accessing or using any Services you also represent that you have the legal authority to accept the Terms on behalf of yourself and any party you represent in connection with your use of any Services. If you do not agree to the Terms, you are not authorized to use any Services.
```
Comment (estradatyjimmytsui) [inline website TOS]: s

**Comment by medo05** [comment]
d470016

**Comment by medo05** [comment]
d470016

**Comment by kengraWin** [comment]
Hi

**Review by cburnett9213 — APPROVED** [review]

**Review by khoangtrong — APPROVED** [review]

**Review by sensesuksawat — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by LillianahPulido** [comment]
Want to used thos

**Pull request closed without merging**